### PR TITLE
feat(observability-lib): multiple alert rules can be attached to panel

### PR DESCRIPTION
**Pull request by Atrax1**
Multiple alert rules can be attached to a single panel

### CODE REVIEW

**Review by jmank88 — APPROVED** [review]

File: observability-lib/dashboards/core-node/component.go [hunk 1]
```diff
@@ -373,47 +418,10 @@ func headlines(p *Props) []*grafana.Panel {
...
-							Params: []float64{90},
-							Type:   grafana.TypeThresholdTypeLt,
-						},
-					},
-				},
-			},
+		AlertsOptions: []grafana.AlertOptions{
+			healthAverageAlertRule(p, 90, map[string]string{"severity": "info"}),
+			healthAverageAlertRule(p, 70, map[string]string{"severity": "warning"}),
+			healthAverageAlertRule(p, 50, map[string]string{"severity": "critical"}),
```
Comment (Atrax1) [inline observability-lib/dashboards/core-node/component.go:424]: So here is that we generate 3 alerts that got different severity depending on the threshold (second param of the function is the threshold)

**Review by skudasov — APPROVED** [review]

File: observability-lib/grafana/dashboard.go [hunk 2]
```diff
@@ -103,9 +104,11 @@ func (db *Dashboard) DeployToGrafana(options *DeployOptions) error {
 			alert.FolderUID = folder.UID
 			alert.Annotations["__dashboardUid__"] = *newDashboard.UID
 
-			panelId := panelIDByTitle(db.Dashboard, alert.Title)
+			panelId := panelIDByTitle(db.Dashboard, alert.Annotations["panel_title"])
+			// we can clean it up as it was only used to get the panelId
+			delete(alert.Annotations, "panel_title")
 			if panelId != "" {
-				alert.Annotations["__panelId__"] = panelIDByTitle(db.Dashboard, alert.Title)
+				alert.Annotations["__panelId__"] = panelId
```
Comment (patrickhuie19) [inline observability-lib/grafana/dashboard.go:111]: why this `__` prefix + suffix combo?

File: observability-lib/dashboards/core-node/component.go (hunk 1 above)
Comment (patrickhuie19) [reply observability-lib/dashboards/core-node/component.go:424]: Very necessary, nice

**Review by patrickhuie19 — APPROVED** [review]

File: observability-lib/grafana/dashboard.go (hunk 2 above)
Comment (Atrax1) [reply observability-lib/grafana/dashboard.go:111]: its internal grafana API that requires this

**Pull request merged**